src/pages/document/[id].tsx

```diff
@@ -168,29 +169,39 @@ const FamilyPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({ pa
             )}
           </section>
 
+          <section className="mt-8" data-cy="top-documents">
+            <DocumentSearchForm
+              placeholder={`Search the full text of the ${page.title}`}
+              handleSearchInput={handleSearchInput}
+              input={router.query[QUERY_PARAMS.query_string] as string}
+              featuredSearches={EXAMPLE_SEARCHES}
+              showSuggestions
+            />
+          </section>
+
           <section className="mt-8">
-            <h2 className="text-2xl">Main documents</h2>
+            <h2 className="text-base">Main document{mainDocuments.length > 1 ? "s" : ""}</h2>
             <div data-cy="main-documents">
-              {mainDocs.map((doc) => (
+              {mainDocuments.map((doc) => (
                 <FamilyDocument matches={getDocumentMatches(doc.slug)} document={doc} key={doc.import_id} status={status} />
               ))}
             </div>
           </section>
 
-          {otherDocs.length > 0 && (
+          {otherDocuments.length > 0 && (
             <>
               <section className="mt-8">
-                <h2 className="flex items-center gap-2 text-2xl">
-                  Related documents{" "}
+                <h2 className="flex items-center gap-2 text-base">
+                  Other documents in this entry{" "}
```

Comment:
We've used this phrase before - worth pulling out as a constant?

Reply:
I don't think yet, this will come when we do the internationalisation IMO